src/functionalTest/java/uk/gov/hmcts/reform/amapi/functional/HealthCheckTest.java

```diff
@@ -1,30 +1,19 @@
 package uk.gov.hmcts.reform.amapi.functional;
 
-import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
-import org.junit.Test;
 import org.junit.jupiter.api.Tag;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Value;
-import uk.gov.hmcts.reform.amapi.functional.client.AmApiClient;
+import org.junit.jupiter.api.Test;
 
+import static io.restassured.RestAssured.get;
 import static org.hamcrest.core.IsEqual.equalTo;
 
-@RunWith(SpringIntegrationSerenityRunner.class)
-@SuppressWarnings({"PMD.JUnitTestsShouldIncludeAssert"})
-public class HealthCheckTest {
-
-    public AmApiClient amApiClient;
-
-    @Value("${TEST_URL:http://localhost:3704}")
-    protected String accessUrl;
+@SuppressWarnings("PMD")
```

Comment:
please avoid suppressing PMD, be more specific if you need to suppress something

Reply:
Renamed method